index.js

```diff
@@ -0,0 +1,7 @@
+const Web3Data = require('./src/web3data')
+// eslint-disable-next-line no-undef
+if (typeof window !== 'undefined' && typeof window.Web3Data === 'undefined') {
```

Comment:
@taylorjdawson this usually is generated or setup differently, please see:
https://github.com/ethers-io/ethers.js/blob/master/dist/ethers.js#L1